lib/licensee/project.rb

```diff
@@ -2,17 +2,6 @@ class Licensee
   class Project
     attr_reader :repository
 
-    # Array of file names to look for potential license files, in order
-    # Filenames should be lower case as candidates are downcased before comparison
-    LICENSE_FILENAMES = %w[
```

Comment:
This is the question I asked in chat... if regex was read left to right. If so, then I believe LICENSE should win over COPYING which should win over COPYRIGHT.